contrib/fonttools/showttf.c

```diff
@@ -2887,6 +2887,7 @@ static void gdefshowligcaretlist(FILE *ttf,int offset,struct ttfinfo *info) {
 		printf( "!!!! Unknown caret format %d !!!!\n", format );
 	    }
 	}
+        free(offsets);
```

Comment:
I just don't know how to do that.
I updated my local repo, then git commit -a --amend
but git push doesn't work
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'https://github.com/serval2412/fontforge.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

If I do "git pull -r", it removes the change.

Moreover, before sending this new patch, I had to scratch my local repo because when I tried to create the new pull request, it included commits from other people.
So just tell me, if you can't merge this one, I'll scratch my local repo again.